samples/meetings-content-bubble/csharp/ContentBubble/Bots/ContentBubbleBot.cs

```diff
@@ -76,23 +137,78 @@ private async Task HandleActions(ITurnContext<IMessageActivity> turnContext)
                         Notification = new
                         {
                             AlertInMeeting = true,
-                            ExternalResourceUrl = $"https://teams.microsoft.com/l/bubble/{ _config["MicrosoftAppId"] }?url=" +
+                            ExternalResourceUrl = $"https://teams.microsoft.com/l/bubble/{_config["MicrosoftAppId"]}?url=" +
                                                   HttpUtility.UrlEncode($"{_config["BaseUrl"]}/ContentBubble?topic={agendaItem.Topic}") +
                                                   $"&height=270&width=250&title=ContentBubble&completionBotId={_config["MicrosoftAppId"]}"
                         }
                     };
                     await turnContext.SendActivityAsync(activity);
                     break;
                 case "SubmitFeedback":
-                    var submitFeedback = Newtonsoft.Json.JsonConvert.DeserializeObject<SubmitFeedbackAction>(turnContext.Activity.Value.ToString());
+                    var submitFeedback = JsonConvert.DeserializeObject<SubmitFeedbackAction>(turnContext.Activity.Value.ToString());
                     var item = _agenda.AgendaItems.First(a => a.Id.ToString() == submitFeedback.Choice);
                     await turnContext.SendActivityAsync($"{turnContext.Activity.From.Name} voted **{submitFeedback.Feedback}** for '{item.Topic}'");
                     break;
+                case "SendNotification":
+                    try
+                    {
+                        var actionSet = JsonConvert.DeserializeObject<ActionBase>(turnContext.Activity.Value.ToString());
+                        var selectedMembers = actionSet.Choice;
+                        var recipients = JsonConvert.SerializeObject(selectedMembers.Split(","));
+                        var pageUrl = JsonConvert.SerializeObject(_config["BaseUrl"] + "/SendNotificationPage");
+                        var meetingId = turnContext.Activity.TeamsGetMeetingInfo()?.Id ?? throw new InvalidOperationException("This method is only valid within the scope of a MS Teams Meeting.");
+
+                        // Notification payload for meeting target notification API.
+                        string notificationPayload = @"{
+                                 ""type"": ""targetedMeetingNotification"",
+                                 ""value"": {
+                                 ""recipients"": " + recipients + @", 
```

Comment:
I really like the example bot's sceanrio btw! 
(1) Returning the list or participants who are in the meeting 
(2) Allowing a user to choose some participants among them to send targeted in-meeting notifications. 
(3) And each targeted in-meeting notifciation shows a task module. 

I think this whole scenario captures the point of the feature really well. Thank you! 🙏🎉👍

Reply:
Thank You !!